styles.css

```
@@ -0,0 +1,934 @@
+  letter-spacing: 0.18em;
+  text-transform: uppercase;
+  color: var(--cyan);
+}
+
+.waveBanner.show {
+  opacity: 1;
+  transform: translateX(-50%) translateY(0);
+  animation: pulseBanner 0.8s ease;
+}
+
+@keyframes pulseBanner {
```

Stylelint (17.3.0)

[error] Expected keyframe name "pulseBanner" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)